src/graphx/graphx.h

```diff
@@ -1063,19 +1081,21 @@ typedef enum {
 
 /**
  * Sets the text foreground color.
+ * Note that if you haven't called
  *
  * @param color Color index to set.
  * @returns Previous text foreground color index.
  * @note Default text foreground color index: 0.
+ * @note Default text background color index is 255, so if you don't change it and try to draw with color 255, nothing will happen.
```

Comment:
perhaps clarification on how to do this properly would be good

Reply:
Added